x-pack/plugins/infra/public/components/waffle/lib/darken.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import Color from 'color';
+
+export function darken(color: string, percent: number) {
```

Comment:
We already have [polished](https://polished.js.org/) as a styled-components helper library. It contains a [darken](https://polished.js.org/docs/#darken) function that can directly be used in the template tags.

Reply:
I was unaware, I will change it to use that.